lib/soapy_cake/helper.rb

```diff
@@ -41,5 +41,9 @@ def const_lookup(type, key)
         fail ArgumentError, "#{key} is not a valid value for #{type}"
       end
     end
+
+    def future_expiration_date
+      Date.today + (365 * 30)
```

Comment:
A little comment explaining the significance of that date would be nice to have.  Is it just an arbitrary date far into the future?